### PR TITLE
feat(csi-128): updated sdk-standard-components with axios impl.; fixed unit-tests

### PR DESCRIPTION
**Pull request by geka-evk**
1. Updated _sdk-standard-components_ with **axios** as http client;
2. Added retry mechanism
3. Improved logging to follow context-logger output format
image

### CODE REVIEW

File: modules/api-svc/package.json [hunk 1]
```diff
@@ -67,12 +67,12 @@
...
     "@mojaloop/central-services-logger": "^11.5.1",
-    "@mojaloop/central-services-metrics": "^12.0.8",
-    "@mojaloop/central-services-shared": "18.11.0",
+    "@mojaloop/central-services-metrics": "^12.1.0",
+    "@mojaloop/central-services-shared": "18.11.1",
     "@mojaloop/event-sdk": "^14.1.1",
     "@mojaloop/ml-schema-transformer-lib": "2.3.4",
     "@mojaloop/sdk-scheme-adapter-private-shared-lib": "workspace:^",
-    "@mojaloop/sdk-standard-components": "^19.4.1",
+    "@mojaloop/sdk-standard-components": "19.5.0-snapshot.12",
```
Comment (kalinkrustev) [inline modules/api-svc/package.json:75]: putting here a reminder to merge @mojaloop/sdk-standard-components first

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-scheme-adapter",
-  "version": "23.6.0-iso.3",
+  "version": "23.7.0-snapshot.1",
```
Comment (kalinkrustev) [inline package.json:3]: this must be restored to 23.6.0-iso.3 before merging

**Review by kalinkrustev — CHANGES_REQUESTED** [review]
I do not know very well the SDK, leaving some generic comments

File: package.json (hunk 2 above)
Comment (geka-evk) [reply package.json:3]: Shouldn't it be done automatically after merging into `minor/iso`?

Comment (kalinkrustev) [reply package.json:3]: Yes, you are right, but then we will need to update the CD to track the new range
See https://github.com/infitx-org/profile-cd/blob/main/updatecli.d/sdk-scheme-adapter.yaml

File: modules/api-svc/package.json (hunk 1 above)
Comment (geka-evk) [reply modules/api-svc/package.json]: done

File: package.json (hunk 2 above)
Comment (geka-evk) [reply package.json]: done

**Pull request merged**